htdocs/product/fournisseurs.php

```diff
@@ -828,6 +829,11 @@ function update_price_from_multicurrency() {
 						print price2num($productfourn->fourn_remise_percent).'%';
 						print '</td>';
 
+						// Date
+						print '<td align="right">';
+						print dol_print_date($productfourn->date_modification, "%d/%m/%Y %H:%M");
```

Comment:
To be compatible with all date format for languages, you can use here 
print dol_print_date($productfourn->date_modification, "dayhour");
instead of 
print dol_print_date($productfourn->date_modification, "%d/%m/%Y %H:%M");